### PR TITLE
Remove jquery usage in install.js

### PR DESCRIPTION
**Pull request by parlough**
Also removes some code and functionality that is no longer used.

Contributes to https://github.com/dart-lang/site-www/issues/3849

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Review by atsansone — CHANGES_REQUESTED** [review]
@parlough : I would recommend replacing the channel descriptions with more parallel and concise language. Suggestion included.

File: src/get-dart/index.md [hunk 1]
```diff
@@ -86,19 +86,19 @@ Dart supports the following macOS versions as of November 2022:
 The Dart SDK has three release channels:
 
 * **Stable** channel: **stable releases**, updated roughly every three months;
```
[the remark this answers is not in the document]
Comment (parlough) [reply src/get-dart/index.md:88]: I would rather keep those changes separate from the JS updates here. I'll open a follow-up PR.

[the remark this answers is not in the document]
Comment (atsansone) [reply src/get-dart/index.md:88]: Done as https://github.com/dart-lang/site-www/issues/4414.

**Review by atsansone — APPROVED** [review]
LGTM for original scope of changes.

File: src/get-dart/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (parlough) [reply src/get-dart/index.md:88]: Thanks!

**Pull request merged**